airflow/api_fastapi/core_api/routes/public/backfills.py

```diff
@@ -149,7 +153,10 @@ def unpause_backfill(backfill_id, session: SessionDep) -> BackfillResponse:
             status.HTTP_409_CONFLICT,
         ]
     ),
-    dependencies=[Depends(action_logging())],
+    dependencies=[
+        Depends(action_logging()),
```

Comment:
Why some endpoints have `Depends(action_logging())` and some have not?

Reply:
I'm not sure 🤔 I just keep the existing ones.

Reply:
Thanks for clarification 🙏

Reply:
I still think we are missing pause / unpause and cancel @vatsrahul1001 is working on a PR.